README.md

```diff
@@ -136,6 +139,38 @@ OktaAuth.login(username: "user@example.com", password: "password").start(view: s
 }
 ```
 
+## Sign Out from Okta
+
+You can start the Sign Out flow by simply calling `signOutFromOkta`. This method will end the user's Okta session in the browser.
+
+```swift
+OktaAuth.signOutFromOkta().start(view: self)
+.then {
+    // Clean tokenManager
+}
+.catch { error in
+    // Error
+}
+```
+
+**Note**: *This method does not clear tokens stored locally, neither revoke them. You must call `signOutLocally` afterwards to do that.*
+
+## Sign Out Locally
```

Comment:
nit: Please remove this section due to our plan to remove the `clearTokens` method from this PR.

Reply:
Done.